rider-fsharp/src/main/java/com/jetbrains/rider/plugins/fsharp/services/fsi/FsiProcessHandler.kt

```diff
@@ -0,0 +1,28 @@
+package com.jetbrains.rider.plugins.fsharp.services.fsi
+
+import com.intellij.execution.process.OSProcessHandler
+import com.intellij.openapi.util.Key
+import java.nio.charset.Charset
+
+class FsiProcessHandler (
+        process: Process, commandLine: String?, charset: Charset?)
```

Comment:
Since we always pass the only particular charset, there is not need to have a parameter for it here.
Please pass UTF-8 charset as an argument directly to the base constructor. Doing that will simplify the `FsiProcessHandler` constructor signature and will reduce a chance of mistake if we ever create instances of this type in different places. We need to have the same charset between fsi process and its process handler and it's currently always UTF-8 that is passed as a command line argument to fsi.

Reply:
Oops, my fault. Fixed.